### PR TITLE
Restructure folders and JS files to manage each page

### PR DESCRIPTION
**Pull request by DKnehi**
Restructure folders and JS files to manage each page

### CODE REVIEW

File: javascript-practice/src/js/app.js [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+import DashboardView from './views/dashboardView';
 
 export default class App {
-  start() {
-    this.controller = new UserController(new UserModel(), new UserView());
+  async start() {
+    const path = window.location.pathname;
+    const model = new UserModel();
+
+    if (path.includes('login')) {
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/app.js]: Path should be constant

File: javascript-practice/src/js/controllers/userController.js [hunk 2]
```diff
@@ -14,22 +14,22 @@ export default class UserController {
...
   handleFormLogin = async (email, password) => {
     try {
       const userData = await this.service.loginUser(email, password);
 
       if (userData) {
-        this.view.redirectPage('dashboard.html');
         localStorage.setItem(LOCAL_STORAGE.FIRST_NAME, userData.firstName);
         localStorage.setItem(LOCAL_STORAGE.LAST_NAME, userData.lastName);
         localStorage.setItem(LOCAL_STORAGE.ROLE, userData.role);
+        this.view.redirectPage('dashboard.html');
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/controllers/userController.js]: Constant too

File: javascript-practice/src/js/app.js (hunk 1 above)
Comment (DKnehi) [reply javascript-practice/src/js/app.js]: i done

File: javascript-practice/src/js/controllers/userController.js (hunk 2 above)
Comment (DKnehi) [reply javascript-practice/src/js/controllers/userController.js]: i done

**Pull request merged**